src/Emogrifier.php

```diff
@@ -843,7 +843,7 @@ private function normalizeStyleAttributesOfAllNodes()
     private function normalizeStyleAttributes(\DOMElement $node)
     {
         $normalizedOriginalStyle = \preg_replace_callback(
-            '/[A-z\\-]+(?=\\:)/S',
+            '/[a-zA-Z\\-]+(?=:)/S',
```

Comment:
Good catch - it happened to work more by luck becuase `:` is before `A` in ASCII.

However, I don't think the correction is quite right - AKAIK property names may also contain underscores or numbers (though I'm not aware of any), if they are classed as [CSS identifiers](https://www.w3.org/TR/CSS21/syndata.html#characters) - the regex used in `HtmlPruner::removeRedundantClassesAfterCssInlined` matches a CSS identifier (after a period).

Reply:
Rebased, changed accordingly and repushed.